robottelo/ui/locators/base.py

```diff
@@ -1279,6 +1279,9 @@
     "rh.prd_expander": (
         By.XPATH, ("//div[@id='ui-tabs-1']//td[contains(.,'%s')]"
                    "/span[@class='expander']")),
+    "rh.ostree_prd_expander": (
+        By.XPATH, ("//div[@id='ui-tabs-7']//td[contains(.,'%s')]"
```

Comment:
Thanks @oshtaier. The locator you suggested is for selecting the content tabs from `RH repositories` page and we are using `href` only for that.

`ui-tab-x` is being used for 'product_expander`, the value of locator is based on the selected content tab. I tried to use class name as well. But that doesn't work. So I left those locators as is. If its really needs to be changed then, I'll handle this another PR.